examples/download_all_raw_recordings_data.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+# pip install --upgrade kachery
+
+import json
+import argparse
+import numpy as np
+import kachery_cloud as ka
+import os
+import spikeforest as sf
+from spikeforest.load_extractors.MdaRecordingExtractorV2.MdaRecordingExtractorV2 import readmda
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Prepare SpikeForest recordings (i.e., populate this repository)")
```

Comment:
Leave out "populate this repository"

Would be great if you could provide a high level description of what this script does / can be used for. What is the end result after running the script?